<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contr√¥le Led et CAN</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>4</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Aucune</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bleu</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rouge</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vert</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Orange</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Toutes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vert + Orange</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bleu + Rouge</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Choix Led:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Demande CAN</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
